ph5/clients/ph5toms.py

```diff
@@ -643,17 +661,25 @@ def create_cut_list(self):
                                                        start_times, station_list, deployment, st_num))
                         elif self.reqtype == "FDSN":
                             # fdsn request
-                            cuts_generator.append(self.create_cut(seed_network, ph5_station, seed_station,
-                                                   start_times, station_list, deployment, st_num))
-                            
-        return itertools.chain.from_iterable(cuts_generator)
+                            stations_list = self.create_cut(seed_network, ph5_station, seed_station,
+                                                   start_times, station_list, deployment, st_num)
+                            for station in stations_list:
+                                cuts_generator.append(station)
+
+        del self.ph5.Experiment_t
+        del self.ph5.Array_t
+        del self.ph5.Event_t
+        self.ph5.clear()
+        self.clear()

```

Comment:
with ph5.clear() they probably aren't necessary. I was deleting them before i started using ph5.clear() 

going back and looking at mprof plots it makes no difference.  We should get rid of those and just use ph5.clear()

ph5.clear() can make a decent difference of 10+MB depending on the experiment size

self.clear() makes a very very tiny difference. Less than a MB. Its basically just clearing all the request parameters after they are no longer needed but before the instance of the class is closed at the end of the program